### PR TITLE
Enabling "View in API"

### LINKED ISSUE #460: Enabling "View in API" button
Request from Slack

> A user was looking for the view API button which is now disabled by default and the user has to enable the Developer tools from the preferences. So, I think we can mention where to enable to view-API if the user wants to see the command from the UI and uses it
>
> I would suggest to update https://ranchermanager.docs.rancher.com/v2.6/pages-for-subheaders/about-the-api to make this clear

### PR DESCRIPTION
**Pull request by sunilarjun**
Updating Capturing API section with intructions on enabling the `View in API` button.

Fixes #460 

## Reminders

- See the [README](../README.md) for more details on how to work with the Rancher docs.

- Verify if changes pertain to other versions of Rancher. If they do, finalize the edits on one version of the page, then apply the edits to the other versions.

- If the pull request is dependent on an upcoming release, make sure to target the release branch instead of `main`.

### CODE REVIEW

File: docs/pages-for-subheaders/about-the-api.md [hunk 1]
```diff
@@ -82,3 +82,5 @@ You can use browser developer tools to capture how the Rancher API is called. Fo
 1. In the Rancher UI, click **Create**. In the developer tools, you should see a new network request with the name `cluster?_replace=true`.
 1. Right-click `cluster?_replace=true` and click **Copy > Copy as cURL.**
 1. Paste the result into any text editor. You will be able to see the POST request, including the URL it was sent to, all of the headers, and the full body of the request. This command can be used to create a cluster from the command line. Note: The request should be stored in a safe place because it contains credentials.
+
+Additionally, you can use the **View in API** link to view captured Rancher API calls for your respective clusters and resources located in the clickable `⋮` from the UI. This feature is not enabled by default. In order to enable, click on your **User Tile** in the top right corner of the UI and from the dropdown click on **Preferences**. Once on the page, navigate to the `Advanced Features` section and click the **Enable "View in API"** button. Once checked, the **View in API** link will now appear under the `⋮` sub-menu on resource pages in the UI.
```
Comment (martyav) [inline docs/pages-for-subheaders/about-the-api.md:86]:
```suggestion
### Enable View in API

You can also view captured Rancher API calls for your respective clusters and resources. This feature is not enabled by default. To enable it: 

1. Click on your **User Tile** in the top right corner of the UI and select **Preferences** from the dropdown menu. 
2. Under the **Advanced Features** section, click **Enable "View in API"**
 
Once checked, the **View in API** link will now appear under the **⋮** sub-menu on resource pages in the UI.
```

File: versioned_docs/version-2.7/pages-for-subheaders/about-the-api.md [hunk 2]
```diff
@@ -82,3 +82,5 @@ You can use browser developer tools to capture how the Rancher API is called. Fo
 1. In the Rancher UI, click **Create**. In the developer tools, you should see a new network request with the name `cluster?_replace=true`.
 1. Right-click `cluster?_replace=true` and click **Copy > Copy as cURL.**
 1. Paste the result into any text editor. You will be able to see the POST request, including the URL it was sent to, all of the headers, and the full body of the request. This command can be used to create a cluster from the command line. Note: The request should be stored in a safe place because it contains credentials.
+
+Additionally, you can use the **View in API** link to view captured Rancher API calls for your respective clusters and resources located in the clickable `⋮` from the UI. This feature is not enabled by default. In order to enable, click on your **User Tile** in the top right corner of the UI and from the dropdown click on **Preferences**. Once on the page, navigate to the `Advanced Features` section and click the **Enable "View in API"** button. Once checked, the **View in API** link will now appear under the `⋮` sub-menu on resource pages in the UI.
```
Comment (martyav) [inline versioned_docs/version-2.7/pages-for-subheaders/about-the-api.md:86]:
```suggestion
### Enable View in API
You can also view captured Rancher API calls for your respective clusters and resources. This feature is not enabled by default. To enable it: 
1. Click on your **User Tile** in the top right corner of the UI and select **Preferences** from the dropdown menu. 
2. Under the **Advanced Features** section, click **Enable "View in API"**
 
Once checked, the **View in API** link will now appear under the **⋮** sub-menu on resource pages in the UI.
```

File: versioned_docs/version-2.8/pages-for-subheaders/about-the-api.md [hunk 3]
```diff
@@ -82,3 +82,5 @@ You can use browser developer tools to capture how the Rancher API is called. Fo
 1. In the Rancher UI, click **Create**. In the developer tools, you should see a new network request with the name `cluster?_replace=true`.
 1. Right-click `cluster?_replace=true` and click **Copy > Copy as cURL.**
 1. Paste the result into any text editor. You will be able to see the POST request, including the URL it was sent to, all of the headers, and the full body of the request. This command can be used to create a cluster from the command line. Note: The request should be stored in a safe place because it contains credentials.
+
+Additionally, you can use the **View in API** link to view captured Rancher API calls for your respective clusters and resources located in the clickable `⋮` from the UI. This feature is not enabled by default. In order to enable, click on your **User Tile** in the top right corner of the UI and from the dropdown click on **Preferences**. Once on the page, navigate to the `Advanced Features` section and click the **Enable "View in API"** button. Once checked, the **View in API** link will now appear under the `⋮` sub-menu on resource pages in the UI.
```
Comment (martyav) [inline versioned_docs/version-2.8/pages-for-subheaders/about-the-api.md:86]:
```suggestion
### Enable View in API
You can also view captured Rancher API calls for your respective clusters and resources. This feature is not enabled by default. To enable it: 
1. Click on your **User Tile** in the top right corner of the UI and select **Preferences** from the dropdown menu. 
2. Under the **Advanced Features** section, click **Enable "View in API"**
 
Once checked, the **View in API** link will now appear under the **⋮** sub-menu on resource pages in the UI.
```

**Review by martyav — CHANGES_REQUESTED** [review]

**Comment by sunilarjun** [comment]
@martyav Thanks for the review Marty! Just applied the changes.

File: versioned_docs/version-2.7/pages-for-subheaders/about-the-api.md [hunk 4]
```diff
@@ -82,3 +82,10 @@ You can use browser developer tools to capture how the Rancher API is called. Fo
 1. In the Rancher UI, click **Create**. In the developer tools, you should see a new network request with the name `cluster?_replace=true`.
 1. Right-click `cluster?_replace=true` and click **Copy > Copy as cURL.**
 1. Paste the result into any text editor. You will be able to see the POST request, including the URL it was sent to, all of the headers, and the full body of the request. This command can be used to create a cluster from the command line. Note: The request should be stored in a safe place because it contains credentials.
+
+### Enable View in API
+You can also view captured Rancher API calls for your respective clusters and resources. This feature is not enabled by default. To enable it: 
+1. Click on your **User Tile** in the top right corner of the UI and select **Preferences** from the dropdown menu. 
+2. Under the **Advanced Features** section, click **Enable "View in API"**
```
Comment (LucasSaintarbor) [inline versioned_docs/version-2.7/pages-for-subheaders/about-the-api.md:89]:
```suggestion

1. Click on your **User Tile** in the top right corner of the UI and select **Preferences** from the dropdown menu. 
2. Under the **Advanced Features** section, click **Enable "View in API"**
```

File: versioned_docs/version-2.8/pages-for-subheaders/about-the-api.md [hunk 5]
```diff
@@ -82,3 +82,10 @@ You can use browser developer tools to capture how the Rancher API is called. Fo
 1. In the Rancher UI, click **Create**. In the developer tools, you should see a new network request with the name `cluster?_replace=true`.
 1. Right-click `cluster?_replace=true` and click **Copy > Copy as cURL.**
 1. Paste the result into any text editor. You will be able to see the POST request, including the URL it was sent to, all of the headers, and the full body of the request. This command can be used to create a cluster from the command line. Note: The request should be stored in a safe place because it contains credentials.
+
+### Enable View in API
+You can also view captured Rancher API calls for your respective clusters and resources. This feature is not enabled by default. To enable it: 
+1. Click on your **User Tile** in the top right corner of the UI and select **Preferences** from the dropdown menu. 
+2. Under the **Advanced Features** section, click **Enable "View in API"**
```
Comment (LucasSaintarbor) [inline versioned_docs/version-2.8/pages-for-subheaders/about-the-api.md:89]:
```suggestion

1. Click on your **User Tile** in the top right corner of the UI and select **Preferences** from the dropdown menu. 
2. Under the **Advanced Features** section, click **Enable "View in API"**
```

**Review by LucasSaintarbor — APPROVED** [review]
Small NIT. LGTM 👍

**Review by martyav — APPROVED** [review]

**Pull request merged**